### PR TITLE
Feature private files page

### PR DESCRIPTION
**Pull request by timothyadewale**
I'm requesting a pull request for the private file page and Admin tools page front end. Please review code

### CODE REVIEW

File: AdminTool.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      </select>
+      <br /><br />
+
+      <select>
+        <option value="">    Organization Role(s) &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp;</option>
+        <option value="">Organization Role</option>
+        <option value="">Organization Role</option>
+        <option value="">Organization Role</option>
+      </select>
+      <br /><br />
```
Comment (JoeyKraemer) [inline AdminTool.html:30]: you are not allowed to use <br> tags

File: AdminTool.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+      <button type="submit">Add New<br />User</button>&nbsp&nbsp&nbsp&nbsp;
+      <button type="submit">Confirm<br />Changes</button>
+      <br /><br /> <br /><br /><br /><br />
```
Comment (JoeyKraemer) [inline AdminTool.html:34]: ..

File: AdminTool.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      <br /><br />
+
+      <select>
+        <option value="">    Organization Role(s) &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp;</option>
+        <option value="">Organization Role</option>
+        <option value="">Organization Role</option>
+        <option value="">Organization Role</option>
+      </select>
+      <br /><br />
+
```
Comment (JoeyKraemer) [inline AdminTool.html:31]: delete empty line

File: AdminTool.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      <button type="submit">Add New<br />User</button>&nbsp&nbsp&nbsp&nbsp;
+      <button type="submit">Confirm<br />Changes</button>
+      <br /><br /> <br /><br /><br /><br />
+    </div>
+
```
Comment (JoeyKraemer) [inline AdminTool.html:36]: ..

File: AdminTool.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  </nav>
+  <main>
+    <div class="tableheader">
+      <h1>USERS</h1>
+    </div>
+    <div class="tableheader"> <input style="background-color: grey;" type="text" /></div>
+
```
Comment (JoeyKraemer) [inline AdminTool.html:43]: ..

File: AdminTool.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+        <td>#123456789</td>
+        <td>LoremIpsum</td>
+        <td>LoremIpsum</td>
+        <td>LoremIpsum</td>
+        <td>first.last@gemorsokos.com</td>
+        <td></td>
+        <td></td>
+      </tr>
+    </table>
+    <br /><br /> <br /><br /> <br /><br /> <br />
```
Comment (JoeyKraemer) [inline AdminTool.html:91]: ..

File: privatefilespage.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
+<!doctype html>
```
Comment (JoeyKraemer) [inline privatefilespage.html:1]: indentation is not to the agreed standard

File: privatefilespage.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+				</div>
+				<div class="uploadicons">
+					<ul>
+						<li> <a class="icons" href="#"> <img src="img/Trash.png" alt="trashbutton"> </a> </li>
+						<li> <a class="icons" href="#"> <img src="img/Upload.png" alt="uploadbutton"> </a> </li>
+						<li> <a class="icons" href="#"> <img src="img/Caret up.png" alt="caretbutton"> </a> </li>
+					</ul>
+				</div>
+			</div>
+			<br><br><br><br>
```
Comment (JoeyKraemer) [inline privatefilespage.html:26]: <br> is not allowed

File: privatefilespage.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+						<ul>
+							<li> <a class="icons" href="#"> <img src="img/Trash.png" alt="trashbutton"> </a> </li>
+							<li> <a class="icons" href="#"> <img src="img/Upload.png" alt="uploadbutton"> </a> </li>
+							<li> <a class="icons" href="#"> <img src="img/Caret up.png" alt="caretbutton"> </a> </li>
+						</ul>
+					</div>
+			</div>
+	</main>
+</body>
+</html>
```
Comment (JoeyKraemer) [inline privatefilespage.html:41]: add empty line at the end of the file

Comment (JoeyKraemer) [reply privatefilespage.html:41]: rename images so they are all lowercase and no "spaces" in the filename

File: css/AdminTool.css [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+    vertical-align: middle;
+    background-color: #FFFF;
+}
+
+th, td, tr {
+    padding:8px;
+    text-align: center;
+    border-bottom: 2px solid #B2BCBC;
+    border-top: 1px solid #B2BCBC;
+}
```
Comment (JoeyKraemer) [inline css/AdminTool.css:65]: add empty line at the end of the file

File: css/privatefilespage.css [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+	background-color:#FFFF;
+	box-shadow: 0px 6px 6px #B2BCBC;
+}
+
+ul{
+	list-style-type: none;
+	display: flex;
+}
+
+
```
Comment (JoeyKraemer) [inline css/privatefilespage.css:63]: delete one empty line

**Review by JoeyKraemer — DISMISSED** [review]
Please address all my comments

**Review by beyzaolmez — APPROVED** [review]

**Pull request merged**